win32_event_log/datadog_checks/win32_event_log/win32_event_log.py

```diff
@@ -30,7 +29,8 @@ class Win32EventLogWMI(WinWMICheck):
     def __init__(self, name, init_config, agentConfig, instances=None):
         WinWMICheck.__init__(self, name, init_config, agentConfig, instances=instances)
         # Settings
-        self._tag_event_id = init_config.get('tag_event_id', False)
+        self._submit_all_events = is_affirmative(init_config.get('submit_all_events', True))
```

Comment:
Let's add a comment to make sure we don't change the default to `False` thinking it was a mistake.